core/server/common/src/main/java/alluxio/master/journal/AbstractJournalSystem.java

```diff
@@ -50,9 +51,11 @@ public synchronized void setMode(Mode mode) {
     }
     switch (mode) {
       case PRIMARY:
+        LOG.info("Transitioning from secondary to primary");
```

Comment:
Should this just be `Transitioning from {} to {}, mMode, mode`?

Reply:
updated, thanks